lib/Parser/DocumentParser.php

```diff
@@ -269,6 +284,18 @@ private function parseLine(string $line): bool
                         $separatorLineConfig = $this->tableParser->parseTableSeparatorLine($line);
 
                         if ($separatorLineConfig === null) {
+                            if ($this->getCurrentDirective() !== null && ! $this->getCurrentDirective()->canApplyToNonBlockContent()) {
+                                // If there is a directive set, it means we are the line *after* that directive
+                                // But the state is being set to NORMAL, which means we are a non-indented line.
+                                // Some special directive (like class) allow their content to be non-indented.
```

Comment:
```suggestion
                                // Some special directives (like class) allow their content to be non-indented.
```

Reply:
you're good at catching my many typos - thanks :)